packages/edit-post/src/components/header/header-toolbar/index.js

```diff
@@ -135,7 +135,9 @@ function HeaderToolbar() {
 					) }
 					showTooltip={ ! showIconLabels }
 				>
-					{ showIconLabels && __( 'Add' ) }
+					{ showIconLabels && ! isInserterOpened
```

Comment:
It seems we need to add parenthesis
```
{ showIconLabels && ( ! isInserterOpened ? __( 'Add' ) : __( 'Close' ) ) }
```

Reply:
Oh, whoops. Miscalculated whether they were necessary here. Sorry, and thanks for the catch!
Updated in most recent commit.